### PR TITLE
Create impersonation_expensify.yml

### PR DESCRIPTION
**Pull request by aidenmitchell**
# Description

Recent increase in Expensify impersonation.

# Associated samples

- https://platform.sublime.security/messages/2beab4d019a0bc277b922b82672237c34e9cc0a689305f54dee7bdc0ce6b03e9

## Associated hunts
- https://platform.sublime.security/hunts/9f7dfdb2-70be-4113-b63a-d53f86860978

### CODE REVIEW

File: detection-rules/impersonation_expensify.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    strings.ilike(sender.display_name, '*expensify*')
+    or strings.ilike(sender.email.email, '*expensify*')
+    or strings.ilevenshtein(sender.email.domain.sld, 'expensify') <= 1
+  )
+  // negating listservs
```
Comment (zoomequipd) [inline detection-rules/impersonation_expensify.yml:12]: how does this impact messages that land in a google group?

Comment (aidenmitchell) [reply detection-rules/impersonation_expensify.yml:12]: This wouldn't flag on messages delivered via google group, IIRC we've seen issues in the past and are waiting for better handling of groups in the platform.

**Pull request closed without merging**